packages/tabs/spec/event-helpers.js

```diff
@@ -1,108 +1,105 @@
-const buildMouseEvent = (type, target, {which, ctrlKey, relatedTarget} = {}) => {
-  const event = new MouseEvent(type, {bubbles: true, cancelable: true})
-  if (which != null) {
-    Object.defineProperty(event, 'which', {
-      get () {
-        return which
-      }
-    })
+
+const buildMouseEvent = (type, target, param) => {
+  if (param === null) {
+    param = {};
   }
```

Comment:
```suggestion
  if (param == null) {
    param = {};
  }
```

I feel like this should be loose equality (`==`), not strict equality (`===`).

`undefined == null` --> `true`. `undefined === null` --> `false`.

(This suggestion would match a similar snippet with loose equals (`==`) on line 52.)

Reply:
Alternate suggestion, to normalize all non-object inputs to objects, but wouldn't match line 52 anymore:

```suggestion
  if (typeof param !== 'object') {
    param = {};
  }
```

Reply:
Last alt suggestion: `if (param == undefined) {`.

I think this is the most direct and to the point. `null` isn't really expected input (but this still handles `null`), emphasizing `undefined` makes it more obvious what this is about, IMO.